content/en/docs/setup/best-practices/certificates.md

```diff
@@ -56,23 +56,36 @@ Required CAs:
 | etcd/ca.crt,key        | etcd-ca                   | For all etcd-related functions   |
 | front-proxy-ca.crt,key | kubernetes-front-proxy-ca | For the [front-end proxy][proxy] |
 
+On top of the above CAs, it is also necessary to get a public/private key pair for service account management, `sa.key` and `sa.pub`.
```

Comment:
What's the impact for a cluster / its operator if these aren't available?

Reply:
> What's the impact for a cluster / its operator if these aren't available?

they are essential.
the public key is fed to the api-server and a private key to the controller-manger.

Reply:
> they are essential.

or rather, the alternative is not a best practice:
https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/

```

--service-account-key-file stringArray
--
  | File containing PEM-encoded x509 RSA or ECDSA private or public keys, used to verify ServiceAccount tokens. The specified file can contain multiple keys, and the flag can be specified multiple times with different files. If unspecified, --tls-private-key-file is used. Must be specified when --service-account-signing-key is provided
```

Reply:
I'd like the reader to understand that setting those is good practice and maybe to be able to read more on a different page.
(@fabriziopandini that could be left for a separate issue, if you favor that approach)

Reply:
I'm ok with expanding the topic, even though might be someone from sig/auth is more appropriate than me for explaining the pro and the cons of different options

Reply:
I'd suggest expanding on this in the service account docs since these aren't necessarily certs.

Reply:
@mikedanese Something for a `whatsnext` section perhaps?